gqcp/tests/Operator/SecondQuantized/USQOneElectronOperator_test.cpp

```diff
@@ -18,10 +18,11 @@
 #define BOOST_TEST_MODULE "USQOneElectronOperator"
 
 #include <boost/test/unit_test.hpp>
+#include <math.h>
```

Comment:
In tests, there's a different order of includes:
1. The necessary boost test includes.
2. GQCP includes
3. Third-party includes (Eigen/...)
4. STL

So, if you're using `<math.h>` inside this test file, it should go at the end.